### PR TITLE
Implemented the useful parts of the Accounts API

### PR DESCRIPTION
**Pull request by Mossman1215**
Hi,
I've implemented the accounts API in the same format as the base sites commands we were working on before.
I only picked the immediately useful endpoints to implement.
Feedback would be great.

### CODE REVIEW

File: src/HttpClient.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ function ($retries, $request, $response, $exception) {
                 if ($exception instanceof ConnectException) {
                     return true;
                 }
-                if ($response && in_array($response->getStatusCode(), $this->retryStatusCodes, true)) {
+                if ($response && \in_array($response->getStatusCode(), $this->retryStatusCodes, true)) {
```
Comment (halkyon) [inline src/HttpClient.php]: `\` is unnecessary

File: src/Command/SubAccountCreateCommand.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->client->Accounts()->add($this->accountName);
```
Comment (halkyon) [inline src/Command/SubAccountCreateCommand.php]: should be lowercase `accounts`

File: src/Client.php [hunk 3]
```diff
@@ -75,6 +76,14 @@ public function sites()
         return new SitesApi($this);
     }
 
+    /**
+     * @return AccountsApi
+     */
+    public function Accounts()
```
Comment (halkyon) [inline src/Client.php]: should be lowercase `accounts`

File: src/HttpClient.php (hunk 1 above)
Comment (Mossman1215) [reply src/HttpClient.php]: ah it created a php cs error in scrutinizer which is the only reason i included it.

**Comment by Mossman1215** [comment]
should be all fixed now :P

File: src/Client.php [hunk 4]
```diff
@@ -75,6 +76,14 @@ public function sites()
         return new SitesApi($this);
     }
 
+    /**
+     * @return accountsApi
+     */
+    public function accounts()
+    {
+        return new accountsApi($this);
```
Comment (halkyon) [inline src/Client.php]: `AccountsApi`? (case)

**Pull request merged**